recipe/meta.yaml

```diff
@@ -59,3 +61,5 @@ extra:
   recipe-maintainers:
     - Borda
     - williamFalcon
+  skip-lints:
+    - python_build_tool_in_run
```

Comment:
I think I would remove this. We fixed the linter so that it stops complaining when `packaging` is present in the `run` requirements. We just need to update the linter in Prefect.

Reply:
Done